strings/base_collections_map.h

```diff
@@ -43,6 +43,18 @@ namespace winrt::impl
 
 WINRT_EXPORT namespace winrt
 {
+    template <typename K, typename V, typename Compare = std::less<K>, typename Allocator = std::allocator<std::pair<K const, V>>>
+    Windows::Foundation::Collections::IMap<K, V> map_view()
+    {
+        return make<impl::input_map_view<K, V, std::map<K, V, Compare, Allocator>>>(std::map<K, V, Compare, Allocator>{});
+    }
+
+    template <typename K, typename V, typename Compare = std::less<K>, typename Allocator = std::allocator<std::pair<K const, V>>>
```

Comment:
Why `map_view` have two overloads, while `vector_view` has a default param value?

Reply:
Similarly to how single_threaded_vector has only one overload and single_threaded_map has two. I followed that.